Resources/Prototypes/Entities/Structures/Doors/Airlocks/external.yml

```diff
@@ -33,3 +33,8 @@
     sprite: Structures/Doors/Airlocks/Glass/external.rsi
   - type: PaintableAirlock
     group: ExternalGlass
+  - type: Fixtures
+    fixtures:
+      fix1:
+        layer:
+        - GlassAirlockLayer
```

Comment:
doesnt this get rid of the other fields like density and the shape  (you can check with VV i think)? you can probably just inherit both AirlockExternal and AirlockGlass (Analyze both prototypes and their parents and what do they inherit and what they overwrite just to make sure you arent adding unwanted things), same for the other one

Reply:
It does seem to get rid of the other fields. The issue I have found with using the AirlockGlass as a parent is that AirlockExternal is a child of the base Airlock. The AirlockExternal has the wrong collision layers while AirlockGlass has different sounds. This results in the door working either as an External or Glass airlock when parenting them both (depending on the order).  I ended up setting the AirlockExternal as the parent and the fixtures that AirlockGlass to the AirlcokExternalGlass.